Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -2578,7 +2911,7 @@ void    pfJournalBook::IMoveMovies( hsGMaterial *source, hsGMaterial *dest )
 
 void    pfJournalBook::IDrawMipmap( pfEsHTMLChunk *chunk, uint16_t x, uint16_t y, plMipmap *mip, plDynamicTextMap *dtMap, uint32_t whichDTMap, bool dontRender )
 {
-    plMipmap *copy = new plMipmap();
+    auto copy = std::make_unique<plMipmap>();
```

Comment:
`hsRef` would be better here, because `plMipmap` extends `hsRefCnt`. (I know the previous code didn't care and `delete`d it directly, but that seems like a risky thing that we should stop doing.)